src/client/pages/Embark/index.tsx

```diff
@@ -245,23 +245,28 @@ interface AngelVariables {
 const useInitQuoteCart = ({ skip = false }) => {
   const quoteCartId = useOnboardingQuoteCartId()
   const [addCampaignCode] = useAddCampaignCodeMutation()
+  const { search } = useLocation()
+  const searchParams = useMemo(() => new URLSearchParams(search), [search])
```

Comment:
If you move search string handling inside initQuoteCart, you won't need useMemo for searchParams.  Less code and slightly more efficient too

Reply:
Great, thanks 👍